python/foglamp/services/south/ingest.py

```diff
@@ -242,13 +246,14 @@ class Ingest(object):
         cls._insert_readings_task = asyncio.ensure_future(cls._insert_readings())
         cls._readings_lists_not_full = asyncio.Event()
 
+        cls._payload_events = cls._parent_service._core_microservice_management_client.get_asset_tracker_events()['track']
```

Comment:
What is this for? I don't see any uses of _payload_events being added in this pull request and do not see how it relates to the discarding of readings.

Reply:
@MarkRiddoch yes, cls._payload_events is not related to discarded readings at all but this is a kind of cache that we maintain locally to save network trip when we check and add an asset to asset_tracker table. We just need to create an initial cache from DB though which is the purpose of the above code and which we missed it earlier.